.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: python
+python:
+    - "2.7"
```

Comment:
@clintonb Tested it for 2.7.10. Its mentioned in README.rst that its designed for 2.7.x